src/main/java/javaserver/Responders/FileResponder.java

```diff
@@ -0,0 +1,61 @@
+package javaserver.Responders;
```

Comment:
Package names should be lowercase to avoid conflicts.

http://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html